docs/utilities/middleware_factory.md

```diff
@@ -3,13 +3,22 @@ title: Middleware factory
 description: Utility
 ---
 
+<!-- markdownlint-disable MD043 -->
```

Comment:
This is actually our time to not disable this rule and implement `Getting started` and `Advanced` ;)